### PR TITLE
docs: fix extinct KeyServer.  Added notes for advanced-user

### PR DESCRIPTION
**Pull request by seetamraju**
Replaced keyserver that no longer exists. Added notes for advanced-security-users for future improvements.

Issue #3944
Pull Request documentation/issue-3944

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/getting-started/verify.en.md [hunk 1]
```diff
@@ -11,8 +11,20 @@ The AWS Copilot CLI executables are cryptographically signed using PGP signature
...
+    Notes to **ADVANCED** Users **ONLY**:
+    *   Let's all switch to use HKPS (HKP over TLS)
+    *   `keyserver.ubuntu.com` is only reachable via IPv4.
+    *   Other EXCELLENT Alternative keyservers are:
+        *   `hkps://keys.openpgp.org`
+        *   `hkps://pgp.mit.edu`
+        *   ~~`hkps://subkeys.pgp.net`~~<BR>
+    *   Since the release of GnuPG 2.1.9 (2015-10-09), the `--keyserver` option for `gpg` has been deprecated and users are recommended to use the `--keyserver` in  `~/.gnupg/dirmngr.conf` instead.<BR>See more in https://unix.stackexchange.com/a/683639
+    *   In the future, if you have to switch KeyServers (again), use this command to verify stored-key using this command 1st -> `gpg --keyserver ${KEYSERVER} --verbose --import-options show-only --recv BCE9D9A42D51784F`
+
```
Comment (huanjani) [inline site/content/docs/getting-started/verify.en.md:27]: !!! Note
    `keyserver.ubuntu.com` is only reachable via IPv4.

Comment (huanjani) [inline site/content/docs/getting-started/verify.en.md:27]: !!! Note
    `keyserver.ubuntu.com` is only reachable via IPv4.

Comment (huanjani) [inline site/content/docs/getting-started/verify.en.md]:
```suggestion
!!! Note
    `keyserver.ubuntu.com` is only reachable via IPv4.   
```

**Comment by huanjani** [comment]
Hello, @seetamraju! 
Thank you SO much for discovering that this was out of date, and doing the work to open this PR! We love your suggestions and helpful tips (and we'll keep them for our own reference), but to stay consistent with the formatting and content of our docs, we've made some edits. Again, thank you for your contribution-- we really appreciate it!

File: site/content/docs/getting-started/verify.en.md (hunk 1 above)
Comment (Lou1415926) [inline site/content/docs/getting-started/verify.en.md]: Alternatively, we can keep the notes but move them to an unexpanded drop-down:
```suggestion

    ???- note "Notes on key servers (optional)"

        * Some alternative key servers are:
            *   `hkps://keys.openpgp.org`
            *   `hkps://pgp.mit.edu`
        * To switch to a different key server, verify the stored-key using this command first: `gpg --keyserver ${KEYSERVER} --verbose --import-options show-only --recv BCE9D9A42D51784F`.

```

I removed some points but kept ones that I felt like might be the most useful for users to complete the verification. Thank you very much for listing out the alternative servers! It's very helpful as sometimes some server go down and people have to switch to another.

Comment (Lou1415926) [reply site/content/docs/getting-started/verify.en.md]: Screen Shot 2022-09-12 at 2 07 31 PM

**Review by Lou1415926 — APPROVED** [review]
I have updated the doc per my comment [here](https://github.com/aws/copilot-cli/pull/3967#discussion_r968931641) and approving the change. Feel free to follow up under this PR if you have any question with the change that I proposed!

**Review by huanjani — APPROVED** [review]

**Pull request merged**